library/src/morpheus-master/sass/modules/_toolmenu.scss

```diff
@@ -491,15 +489,12 @@ nav#subSites{
 			&.is-current{
 				position: relative;
 				margin-right: -1px;
+				border-left: $tool-menu-item-selected-border-left;
 
 				.Mrphs-toolsNav__menuitem--link{
 					color: $tool-menu-item-selected-text-color;
 					background: $tool-menu-item-selected-background-color;
 
-					&:before{
```

Comment:
I recommend keeping this rule and moving the whole contents of the :before rule from above (when it was applied to all tool links) to just this current item (so it will apply only when needed). Because it will be positioned absolutely, it should affect the flow of the document or the tool name's position. This should also allow you to achieve your primary goal of this issue: to remove the extra borders for non-current/non-selected tools.

Reply:
That's fine just submit your changes, I won't have time to work on this or test this again for awhile. I'll just close this PR.